### PR TITLE
added Search Bar and search function to interpret search query

### PR DESCRIPTION
**Pull request by Blandon22**
## Changes Made
Added Search Bar and search Function
### Screenshots
image
image

## Future Steps
Finalize search bar sizing across all screen sizes. Currently, it's using arbitrary proportions for small, medium, and large screens. 
## Checklist

- [] Functionality works and does not implicate other code
- [] Written tests if applicable
- [] Error checking if applicable
- [] Code quality 

#### Issues Resolved

Closes #

### CODE REVIEW

File: carleton-pathways/src/components/searchBar/SearchBar.js [hunk 1]
```diff
@@ -1,7 +1,48 @@
...
+    if (event.key === 'Enter') {
+
+      handleSearch();
+    }
+  };
+
+
   return (
-    <div>SearchBar</div>
+
```
Comment (kunwar45) [inline carleton-pathways/src/components/searchBar/SearchBar.js:19]: Add a empty react tag of `<>` here.

File: carleton-pathways/src/components/searchBar/SearchBar.js [hunk 2]
```diff
@@ -1,7 +1,48 @@
...
+      onKeyDown={handleKeyDown}
+      />
+      <style jsx>{`
+        input[type="search"]::-webkit-search-cancel-button {
+          display: none;
+        }
+      `}</style>
+    
+  </div>
+</div>
```
Comment (kunwar45) [inline carleton-pathways/src/components/searchBar/SearchBar.js:46]: add a closing react tag of `</>` here

**Review by kunwar45 — CHANGES_REQUESTED** [review]
Amazing! Just change the file name from `SearchBar.js` to `SearchBar.jsx`. jsx works better for react

**Review by Iman1121 — APPROVED** [review]
Nice work!

**Review by kunwar45 — APPROVED** [review]

**Pull request merged**